.gitattributes

```diff
@@ -0,0 +1,3 @@
+/.gitattributes     export-ignore
+/README.md          export-ignore
+/Resources/meta     export-ignore
```

Comment:
It's the convention in Symfony bundles. As you can see in [their best practices for bundles](http://symfony.com/doc/current/cookbook/bundles/best_practices.html#directory-structure), all the directory names in `Resources` are lowercased.